SPEC.md

```diff
@@ -1,25 +1,34 @@
 # Knyle Style Sheets
 
-Inspired by [TomDoc](http://tomdoc.org), KSS attempts to provide a methodology for writing maintainable, documented CSS within a team. Specifically, KSS is a documentation specification and styleguide format. It is **not** a preprocessor, CSS framework, naming convention, or specificity guideline.
+KSS attempts to provide a methodology for writing maintainable, documented CSS within a team. Specifically, KSS is a documentation specification and style guide format. It is **not** a preprocessor, CSS framework, naming convention, or specificity guideline.
+
+However, there are several software projects that provide preprocessors that read and parse KSS-compliant documentation to produce full-blown style guides in HTML. Most of those implementations provide:
+
+1. a tool to generate a style guide by parsing CSS source files to find KSS docs, and
+2. a language-dependent API that represents the style guide as a data structure.
+
+This specification does not discuss those implementations.
```

Comment:
Yeah, I was starting to feel like this was too heavy when I wrote it. And I agree it's too early.